common/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "private": true,
+  "name": "univaf-common",
+  "version": "0.1.0",
+  "description": "Common tooling for other parts of UNIVAF.",
+  "exports": {
+    ".": "./src/index.js",
+    "./*": "./src/*.js"
+  },
+  "types": "./dist/src/index.d.js",
+  "typesVersions": {
+    "*": {
+      ".": [
+        "./dist/src/index.d.js"
+      ],
+      "./*": [
+        "./dist/src/*"
+      ]
+    }
+  },
```

Comment:
Note to self for later: I think there’s a way to put the types inside the `exports` object, but I was having trouble with path resolution when I first tried it. Now that I have paths fixed, it might be worth revisiting. See https://devblogs.microsoft.com/typescript/announcing-typescript-4-7/